src/MiniProgressBars.jl

```diff
@@ -74,7 +74,7 @@ function show_progress(io::IO, p::MiniProgressBar; termwidth=nothing, carriagere
     max_progress_width = max(0, min(termwidth - textwidth(p.header) - textwidth(progress_text) - 10 , p.width))
     n_filled = floor(Int, max_progress_width * perc / 100)
     partial_filled = (max_progress_width * perc / 100) - n_filled
-    n_left = max_progress_width - n_filled - 1
+    n_left = max_progress_width - n_filled
```

Comment:
Is line 91 unconditionally true now?

Reply:
No it needs to stay for when the bar is at 100%. Without it, an extra unwanted `'╺'` gets printed and then line 97 errors for trying to repeat a string -1 times.